samples/taglist.cpp

```diff
@@ -58,6 +43,26 @@ int main(int argc, char* argv[])
                     break;
                 }
 
+                if (item == "all" || item == "ALL" ) {
+                    const GroupInfo* groupList = ExifTags::groupList();
+                    if (groupList) {
+                        while (groupList->tagList_) {
+                            std::ostringstream tags;
+                            ExifTags::taglist(tags,groupList->groupName_);
+                            // https://en.cppreference.com/w/cpp/string/basic_string/getline
+                            std::istringstream input(tags.str()) ;
+                            for (std::string line,comma(","); std::getline(input, line); ) {
```

Comment:
[suggestion] Could this be replaced to this version?

```cpp
std::string line;
while (std::getline(input, line)) {
  std::cout << groupList->groupName_ << "."
  << (item == "all" ? line.substr(0,line.find(",")) : line)
  << std::endl;
}
```

I find that line 54 difficult to read.